### ISSUE TITLE
Encoding doesn't stay at source fps

### ISSUE THREAD

**Issue by FlappyFalcon** [ekisu/mpv-webm#84]
Windows 10 Version 1909 (OS Build 1863.778)
[log.txt](https://github.com/ekisu/mpv-webm/files/4607408/log.txt)

As the title says when I encode a file that has a source fps of 23.976 (24000/1001), there's always an error. I added `--vf=fps=\"24000/1001\"` at the end of additional flags which looks like this now:

	`additional_flags = "--vf-add=format=yuv420p --ovcopts-add=profile=high --ovcopts-add=level=4 --vf=fps=\"24000/1001\"",`

When I remove `--vf=fps=\"24000/1001\` it encodes but the output fps ends up being like 24.173 or something else, which isn't what I want. I was told it's because I need to change `--ovcopts-add=profile=high` to _high10_, but when I did it only encodes at 1080p, and doesn't convert from 10bit to 8bit. 

Basically what I want to do is encode at 480p High Level 4 at 23.976 (24000/1001) fps, but it isn't letting me.

**Comment by ekisu** [comment]
Move `--vf=fps=\"24000/1001\"` to the beginning of the flags.

**Comment by FlappyFalcon** [comment]
> Move `--vf=fps=\"24000/1001\"` to the beginning of the flags.

I did that but it still only outpts to 1080p even though I selected 480p. The bit depth & level conversion worked though. Not sure why it won't resize down to 480.

**Comment by ekisu** [comment]
Oops, change `vf` to `vf-add`. Using `vf` replaces the existing filter chain, which explains why it encoded at 1080p (removed `scale` filter) and with 10-bit (removed `format` filter).

**Comment by FlappyFalcon** [comment]
That worked, thanks
